service-front/module/Application/view/application/general/cookies/index.twig

```diff
@@ -1,44 +1,50 @@
 {% extends 'layout/layout.twig' %}
-
 {%- set pageTitle = 'Cookies' -%}
-
 {% import "application/macros.twig" as macros %}
-
 {% set usageCookies = form.get('usageCookies') %}
-
 {% block content %}
 
     <p>Cookies are files saved on your phone, tablet or computer when you visit a website.</p>
-
     <p>
         We use cookies to store information about how you use the Make a lasting power of attorney service, such as the
         pages you visit.
     </p>
 
-
     {# begin form #}
     {{ form.prepare() ? '' }}
     {{ form().openTag( form )|raw }}
         {{ formElement(form.getCsrf) }}
 
-        <h2 class="heading-large">Cookie settings</h2>
+    <h2 class="heading-large">Change your cookie settings</h2>
 
-        <p>We use two types of cookie. One type is strictly necessary and the other type of cookie is optional.</p>
+    <p>We use two types of cookie. One type is strictly necessary and the other type of cookie is optional.</p>
 
-        <h3 class="heading-medium">Cookies that measure website use</h3>
+    <h3 class="heading-medium">Strictly necessary cookies</h3>
+    <p>These essential cookies allow you to log in to the service. They always need to be on.</p>
 
+    <h3 class="heading-medium">Cookies that measure website use</h3>
+    <p>
+        OPG and Government Digital Services use Google Analytics to measure how you use the service so we can improve it
+        based on user needs. Google Analytics sets cookies that store anonymised information about:
+    </p>
+    <ul class="list list-bullet">
+        <li>what pages you visit and how long you spend on each</li>
+        <li>what you click on while you're visiting the site</li>
+    </ul>
+    <p>We do not allow Google to use or share the data about how you use this site.</p>
+
+    <div class="no-js" hidden>
         <p>
-            OPG and Government Digital Services use Google Analytics to measure how you use the service so we can improve it
-            based on user needs. Google Analytics sets cookies that store anonymised information about:
+            We cannot change your cookie settings at the moment because JavaScript is not running in your browser.
+            To fix this, try:
         </p>
-
         <ul class="list list-bullet">
```

Comment:
These steps need to be carried out in order, so number them with `<ol>` rather than bullet them.

Reply:
Not really, depends on what's going on locally with their browser. These are only meant as a suggestion so they can look at what might be going on & not a definitive solution to what issues they are having locally.